pkg/manifests/template/helm.go

```diff
@@ -160,12 +170,60 @@ func (h *helm) templateHelm(conf *action.Configuration, name, namespace string,
 	if err != nil {
 		return nil, err
 	}
-	var manifests bytes.Buffer
-	_, err = fmt.Fprintln(&manifests, strings.TrimSpace(rel.Manifest))
+
+	manifests := strings.Split(rel.Manifest, "---")
```

Comment:
I'm pretty sure there's a place further up in the code where we're handling these as kubernetes unstructured objects where it'd be more natural to just set the labels explicitly w/o the string parsing here.

Reply:
Yeah perhaps just loop through the result of streamManifests in the `Render` function for the helm templater, which are all just unstructured objects which can be modified in-place

Reply:
Oh, right. I didn't notice that stream manifests is using unstructured already.

Reply:
Fixed.